.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vcpkg"]
+path = vcpkg
+url = https://github.com/microsoft/vcpkg.git
```

Comment:
I am not sure if this should be a submodule or a super build dependency. Not every build will need to have this resource. The GTest repo is another super build dependency that is only a clone and not complied code:
https://github.com/SimpleITK/SimpleITK/blob/master/SuperBuild/External_GTest.cmake

Reply:
Removed. Just follow the official VCPKG installation instructions. You don't need to add any Git submodules.